app/serializers/provider_serializer.rb

```diff
@@ -87,22 +94,8 @@ def type_of_gt12
     @object.ucas_preferences.type_of_gt12_before_type_cast
   end
 
-  attribute :contacts do
-    %w[
-      admin
-      utt
-      web_link
-      fraud
-      finance
-      application_alert_recipient
-    ].map do |type|
-      {
-        type: type,
-        name: "#{type.humanize.titleize} Contact #{@object.provider_code}",
-        email: @object.email&.sub(/.*@/, "#{type}@"),
-        telephone: @object.telephone,
-      }
-    end
+  def application_alert_recipient
+    @object.ucas_preferences.application_alert_email
```

Comment:
The API spec doesn't list this attribute on the `providers` endpoint, so probably not worth exposing it here.

Reply:
Done and refactored the tests so to ensure it only appears once (in contacts not prov pref)